### ISSUE TITLE
The 'show accessibility help' command is associated with the alt + f1 key but that key is used in gnome.

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#92333]
Issue Type: <b>Bug</b>

In gnome the shortcut alt + f1 is used to Switch between overview and desktop.
In VSCode, the same combination is used to activate accessibility in the editor.
Priority ends up being the gnome that handles this key combination preventing it from being used by VSCode.
@isidorn  Would it be possible to use another shortcut instead of alt + f1?

VS Code version: Code - Insiders 1.44.0-insider (1876b51870e30e9ba8f35dc0dcfaf49a3799a8ef, 2020-03-09T20:10:25.732Z)
OS version: Linux x64 5.5.8-arch1-1

**Comment by isidorn** [comment]
@jvesouza of course, we can change the default for Linux. What would be your suggestion of a keybinding?
To not potentially break users muscle memory I would just add an additional keybinding on Linux.
@jvesouza is that keybinding taken only on Gnome or on most Linux platforms?

@alexdima fyi

**Comment by jvesouza** [comment]
I believe that all distros that use gnome will have the shortcut alt + f1 with this function.
The suggestion would be to use alt + shift + f1

**Comment by isidorn** [comment]
@jvesouza great, thanks for feedback. We have pushed this change, you can try it out in vscode insiders and let us know if all works fine now.

**Comment by jvesouza** [comment]
@isidorn Today I downloaded a new version but save errors on my part, nothing has changed in relation to this subject.


Version: 1.44.0-insider
Commit: 1df23554b2e3d5f1efc6fbc76ee61d3f7f186c6d
Date: 2020-03-12T05:34:53.829Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Linux x64 5.5.8-arch1-1

**Comment by isidorn** [comment]
@jvesouza I missed on spot. I only did it for the standalone editor, not for the workbench.
Now I pushed a proper fix. So tomorrow it should be good. Thanks for reporting the issue.

**Comment by jvesouza** [comment]
@isidorn  It is fixed, thanks.

**Comment by isidorn** [comment]
Great, adding verified label.

**Comment by jvesouza** [comment]
@isidorn I found that if editor.accessibilitySupport is not set to on, the message read by orca suggests that the alt + f1 key combination should be used. It seems to me that the use of the combination shift + alt + f1 should be suggested.
It is important to note that pressing the combination shift + alt + f1 produces the expected result.

**Comment by isidorn** [comment]
@jvesouza you are correct. I have pushed a commit to tackle this issue.
